testing_and_setup/compass/ocean/global_ocean/WC14/spin_up/config_test_final_settings.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
+<config case="test_final_settings">
+	<add_link source="../../init/initial_state/initial_state.nc" dest="init.nc"/>
+	<add_link source="../../init/initial_state/graph.info" dest="graph.info"/>
+	<add_link source="../../init/initial_state/init_mode_forcing_data.nc" dest="forcing_data.nc"/>
+	<add_link source="../spin_up1/Restart_timestamp" dest="Restart_timestamp"/>
+	<add_link source="../spin_up1/restarts" dest="restarts"/>
+
+	<add_executable source="model" dest="ocean_model"/>
+
+	<namelist name="namelist.ocean" mode="forward">
+		<template file="template_forward.xml" path_base="script_configuration_dir"/>
+		<template file="template_monthly_output.xml" path_base="script_configuration_dir"/>
```

Comment:
@xylar and @mark-petersen this is causing a failure at this stage of the spin up because this line wasn't included in streams below as well, but even doing that ```template_monthly_output.xml``` did not include ```timeSeriesStatsMonthlyRestart```.  I can make these changes, but my question is if we should do this here, since the xml template doesn't appear to be part of this PR.

Reply:
@vanroekel, this was part of the reason for an issue I posted in April: https://github.com/MPAS-Dev/MPAS-Model/issues/525

In my view, any changes you have to make to `config_final_settings.xml` are fair game for this pull request and hopefully would let us close #525.  I would have like to have this addressed in a separate PR but that obviously didn't happen and now is the time.  Sorry for the mess!

Reply:
My suggestion would be to remove `timeSeriesStatsMonthlyOutput` because:
* it doesn't work well unless you run for a full month
* it doesn't work well if you don't start at the beginning of a month
* spin-up isn't always compatible with running for exactly 1 month prior the `test_final_settings` stage.

Reply:
I agree I would prefer to just disable ```timeSeriesMonthly```. I will make the change.